src/status_im/contexts/wallet/connected_dapps/view.cljs

```diff
@@ -25,23 +25,21 @@
                                  {:id   :dapp-disconnect-success
                                   :type :positive
                                   :text (i18n/label :t/disconnect-dapp-success
-                                                    {:dapp    name
-                                                     :account (:name wallet-account)})}]))
+                                                    {:dapp name})}]))
      :on-fail    (fn []
                    (rf/dispatch [:toasts/upsert
                                  {:id   :dapp-disconnect-failure
                                   :type :negative
                                   :text (i18n/label :t/disconnect-dapp-fail
-                                                    {:dapp    name
-                                                     :account (:name wallet-account)})}]))}]))
+                                                    {:dapp name})}]))}]))
```

Comment:
The dapp name might be empty. We should always use `compute-dapp-name` function to get the name of the dapp.

Reply:
done 👍